native/src/main/java/io/ballerina/persist/nodegenerator/BalSyntaxTreeGenerator.java

```diff
@@ -466,6 +701,54 @@ private static Class createClientClass(Entity entity, String className, List<Nod
         delete.addStatement(NodeParser.parseStatement(BalFileConstants.DELETE_RUN_DELETE_QUERY));
         client.addMember(delete.getFunctionDefinitionNode(), true);
 
+        if (!entity.getRelations().isEmpty()) {
+            Function exists = new Function(BalFileConstants.EXISTS);
+            if (keys.size() > 1) {
+                keyString = new StringBuilder();
+                keyString.append("{");
+
+                for (String key : keys.keySet()) {
+                    keyString.append(entity.getEntityName().substring(0, 1).toLowerCase() +
+                            entity.getEntityName().substring(1));
+                    keyString.append(".");
+                    keyString.append(key);
+                    keyString.append(", ");
+                }
+                keyString.append("}");
+
+            } else {
+                keyString = new StringBuilder();
+                keyString.append(entity.getEntityName().substring(0, 1).toLowerCase() +
+                        entity.getEntityName().substring(1));
+                keyString.append(".");
+                for (String key : keys.keySet()) {
+                    keyString.append(key);
+                }
+            }
+
+
```

Comment:
Shall we remove extra lines

Reply:
updated in [fe79bc0](https://github.com/ballerina-platform/persist-tools/pull/23/commits/fe79bc066762c9cbdb981fbfb081b6efea3af0d4)